tests/readers.test.js

```diff
@@ -81,13 +152,17 @@ describe('/readers', () => {
         const reader = readers[0];
         const response = await request(app)
           .patch(`/readers/${reader.id}`)
-          .send({ email: 'miss_e_bennet@gmail.com' });
+          .send({ email: 'miss_e_bennet@gmail.com' })
+          .send({ password: 'password1' });
```

Comment:
can you put these both in the same send by putting both bits of data in one object?